js/repl/loadBabel.js

```diff
@@ -39,13 +48,12 @@ export default function loadBabel(
     return;
   }
 
+  let version = config.version;
```

Comment:
I think the earlier position of this code was better, since the comment explains what this variable is for (`config.version` is passed in dev, which is mentioned in the comment)

Reply:
Didn't actually intend to move it. Have just been iterating on this code.